stl/inc/memory

```diff
@@ -2295,8 +2311,7 @@ public:
 
     template <class _Ty2, enable_if_t<_SP_pointer_compatible<_Ty2, _Ty>::value, int> = 0>
     weak_ptr(weak_ptr<_Ty2>&& _Other) noexcept { // move construct from _Other

```

Comment:
Pre-existing rage at "move construct from other".

Reply:
I'll just drop these comments - they add no value and we don't comment the assigns.